src/com/google/javascript/jscomp/CommandLineRunner.java

```diff
@@ -303,6 +303,12 @@
         "(i.e. input-file-path|input-source-map)")
     private List<String> sourceMapInputs = new ArrayList<>();
 
+    @Option(name = "--apply_input_source_map",
+        hidden = true,
+        usage = "Whether to apply input source maps to the output source map, " +
+        "i.e. have the result map back to original inputs")
+    private final boolean applyInputSourceMap = false;
```

Comment:
This needs `handler = BooleanOptionHandler.class`, otherwise `--apply_input_source_map true` fails.